tests/bench.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
```

Comment:
Although this is supported by gcc and clang, this is not standard, could you use the regular include guard instead?

Also there's probably a license header to include if it's from google benchmark?

Reply:
Fixed!